src/gui/layout/qgslayoutlegendwidget.cpp

```diff
@@ -812,6 +818,24 @@ void QgsLayoutLegendWidget::mMoveUpToolButton_clicked()
   mLegend->endCommand();
 }
 
+void QgsLayoutLegendWidget::expandLegendTree()
+{
+  if ( !mLegend )
+  {
+    return;
+  }
```

Comment:
I see that this check is indeed done in other slots, but they de-reference mLegend, whereas we don't. The check doesn't hurt, but for the sake of keeping the code to the minimum needed, I believe we could also just remove it:

```suggestion
```